src/cmd/top/summary_pane.c

```diff
@@ -152,7 +152,7 @@ static void draw_stats (struct summary_pane *sum)
     }
 
     if (sum->show_details) {
-        int failed = sum->stats.failed;
+        int failed = sum->stats.failed + sum->stats.timeout + sum->stats.canceled;
```

Comment:
As in the JobStats class, maybe a comment stating here that `flux top` reports the total count of unsuccessful jobs in the `failed` stats? A minor suggestion since I guess it is pretty obvious.